paasta_tools/cli/cmds/status.py

```diff
@@ -1465,7 +1473,7 @@ class ReplicaState(Enum):
     UNKNOWN = "Unknown", PaastaColors.yellow
 
     def is_unhealthy(self):
-        return self.name in ["UNREACHABLE", "NOT_RUNNING"]
+        return self.color == PaastaColors.red
```

Comment:
clever :p 

i'm a little worried we'll accidentally break this compared to checking if an element is in an `UNHEALTHY_REPLICA_STATES` set - but that seems kinda unlikely

Reply:
maybe we add a docstring in the ReplicaState enum to make it clear that the color is quite important?